src/lib/supported-node-js-version-check.js

```diff
@@ -0,0 +1,46 @@
+// Ref. https://nodejs.org/en/about/releases
+const nodeVersionToEndOfLifeDateMap = {
+	'12': new Date('2022-04-30'),
+	'14': new Date('2023-04-30'),
+	'16': new Date('2023-09-11'),
+	'18': new Date('2025-04-30'),
+	'20': new Date('2026-04-30')
+};
+
+/**
+ * @param {string} appName
+ * @returns {import("../../typings/metrics").Healthcheck}
+ */
+module.exports = (appName) => {
+	const lastUpdated = new Date();
+
+	const currentDate = new Date();
+
+	const region = process.env.REGION === 'US' ? 'us' : 'eu';
+
+	const nodeVersion = process.versions.node;
+
+	const nodeMajorVersionRegex = new RegExp(/^([0-9]*)\./);
+
+	const nodeMajorVersion = nodeVersion.match(nodeMajorVersionRegex)[1];
```

Comment:
**suggestion:** I'd be tempted to use the [semver](https://www.npmjs.com/package/semver) package here rather than a regular expression. Not essential and non-blocking, but it'd be easier to read and less error-prone.

This module is already included a bunch of times in `package-lock.json` so it wouldn't increase the bundle size or anything if we installed it as a regular dependency.

```suggestion
	const nodeVersion = process.versions.node;
	const nodeMajorVersion = semver.major(nodeVersion);
```

Reply:
This is much cleaner - I have applied this change in commit [Add semver package to extract major version from Node.js version](https://github.com/Financial-Times/n-express/pull/718/commits/97f6b949446e28c402a4af27223c1a89b8a6ee63)